prometheus_hardware_exporter/collector.py

```diff
@@ -349,11 +358,39 @@ def fetch(self) -> List[Payload]:
             logger.error("Failed to fetch current power from ipmi dcmi")
             return [Payload(name="ipmi_dcmi_command_success", value=0.0)]
 
+        get_ps_redundancy_ok, ps_redundancy = self.ipmi_tool.get_ps_redundancy()
+        # Because we fail to get the redundancy config from the server,
+        # Suppose redundancy enable make denominator smaller
+        # and alert is more easy to fire.
+        if not get_ps_redundancy_ok:
+            ps_redundancy = True
+
+        power_capacities = self.lshw.get_power_capacities()
+        maximum_power_capacity = (
+            sum(power_capacities) / len(power_capacities)
+            if ps_redundancy
+            else sum(power_capacities)
```

Comment:
I don't recall ever seeing a server using two power supplies in parallel or in a combined fashion. Is it even possible?

Reply:
It's possible according to the hardware design I know, it's rare case.
If you think we don't need to support this I can remove it.

Reply:
Another issue is if there are two powers with different power capacities, We don't have a way to know which power is in used from the command. Do you think average make sense here?